cypress/integration/ambianic-tests/onboarding.spec.js

```diff
@@ -23,19 +25,29 @@ context('Onboarding Wizard', () => {
     cy.get('[data-cy=continue-to-edge-installation]').click()
   })
 
-  it('It confirms remote request message and request grant', () => {
```

Comment:
@vickywane why did you get rid of the request grant test? It's still a valid scenario we need to cover. Check that the request dialogs show up, etc. Test that the offered request mediums show up as expected: email, message.